### ISSUE TITLE
[Permission] The App ask permission for writing/accessing photos the first time you launch it ...

### ISSUE THREAD

**Issue by FrancyCavallari** [vgwb/Antura#331]
... so it's not clear for the player WHY the app asks for that permission.

Can we delay the request until the Photo Tutorial starts?

**Comment by StefanoCecere** [comment]
no, we can't. since the image gallery plugin is auto inited at boot.
many apps work like this
